tests/apollo/test_skvbc_preexecution.py

```diff
@@ -30,7 +30,7 @@ def start_replica_cmd(builddir, replica_id):
     """
 
     status_timer_milli = "500"
-    view_change_timeout_milli = "10000"
+    view_change_timeout_milli = "5000"
```

Comment:
why 5sec here? With less than 10 seconds I've seen weird view change behaviors. View change can be a slow process (in the order of 10-s of seconds)

Reply:
Should be OK with status_timer_milli of 500ms, but I'll change.